bot/exts/info/information.py

```diff
@@ -243,7 +243,9 @@ async def create_user_embed(self, ctx: Context, user: FetchedMember) -> Embed:
 
         if on_server:
             joined = discord_timestamp(user.joined_at, TimestampFormats.RELATIVE)
-            roles = ", ".join(role.mention for role in user.roles[1:])
+            # The 0 is for excluding the default @everyone role,
+            # and the -1 is for ordering the roles from highest to lowest.
```

Comment:
This makes it seem like you sort them. Just say that it reverses the order.

Reply:
Well, it *is* sorted - from lowest to highest. `-1` makes it highest to lowest. I'll add "instead of the other way around" to the end of the comment to clear any possible confusion.